### DISCUSSION TITLE
How to get the favicon for the PWA to display in OS status bar.

### DISCUSSION THREAD

**Discussion by iconoclasthero** [filips123/PWAsForFirefox#306] [Q&A] [answered]
I was able to get Chrome to display the favicon in the OS (Ubuntu 22.10) status bar but not FF: https://superuser.com/questions/1770852/how-to-display-a-properly-working-favicon-for-icecast2-ubuntu-web-client-site-sp

The icecast icon for the Chrome SSB is immediately to the left of the boxed 2 that represents the FF SSB created by PWAs for Firefox:

![Screenshot from 2023-02-26 20-16-22](https://user-images.githubusercontent.com/535962/221452538-23aaa784-8466-4682-b5b3-071e0ec73811.png)

**Comment by filips123** [comment, marked as answer]
PWAsForFirefox uses icons from `link[rel=icon]` tags in the website document. These tags need to be defined if you want icons to be displayed.

Alternatively, since 2.4.0, you can enter a custom icon URL when installing Icecast (#128).
